Makefile

```diff
@@ -1,7 +1,28 @@
-.PHONY: lint
+VENV=venv
+PIP_LOG=pip-log.txt
```

Comment:
What's the use case for writing pip logs to an extra file rather then to stdout?

Reply:
See https://github.com/openzipkin-contrib/apache-release-verification/pull/9#discussion_r258701658 – I'll resolve this thread to keep this conversation in one place.